lib/spack/llnl/util/filesystem.py

```diff
@@ -1180,6 +1195,27 @@ def remove_if_dead_link(path):
         os.unlink(path)
 
 
+def error_remove_readonly(func, path, exc):
+    """Sets permissions of file pointed to by path
+    to be writeable on Windows if exc denotes an
+    access error and func is removing a file/dir/link
+
+    implements the callback interface for shutil.rmtree
+
+    Note: This is a no-op on non Windows platforms and will
```

Comment:
I think this part of the docstring is now out of date (can be removed).

Reply:
Oh I read more carefully: this *does* say it will throw an error on non-Windows. I think that would mean it isn't a noop though (so in conclusion, can you remove the part that says it's a noop?).

Reply:
Change made!